openedx/features/content_tagging/rules.py

```diff
@@ -88,7 +120,7 @@ def can_change_object_tag(user: User, object_tag: oel_tagging.ObjectTag = None)
 
 
 # Taxonomy
-rules.set_perm("oel_tagging.add_taxonomy", can_change_taxonomy)
+rules.set_perm("oel_tagging.add_taxonomy", can_add_taxonomy)
```

Comment:
This was necessary to handle the difference in taxonomy=None from can_change_taxonomy.